### ISSUE TITLE
IE9 Console Error breaks scripts/Sidebar subnavs

### ISSUE THREAD

**Issue by RachelRVasquez** [JDRF/design-system#378]
Noticed that there is a console error that may be interfering with the sidebar nav in IE9. We don't see the child pages in the subnav in IE9 only (screenshots below).

As for the rest, it's not immediately obvious until we've re-integrated a menu button to expand/collapse the sidebar menu on desktop, but the error will interfere with this as well ( once the PR is merged from https://github.com/JDRF/design-system/issues/147 ).

![screen shot 2016-09-22 at 1 45 03 am](https://cloud.githubusercontent.com/assets/10246844/18738105/8b4a55b0-8066-11e6-8132-6cc7ebe19bc7.png)
![screen shot 2016-09-22 at 1 44 55 am](https://cloud.githubusercontent.com/assets/10246844/18738104/8b4985ea-8066-11e6-820e-fa2cc21df224.png)

**Comment by RachelRVasquez** [comment]
It looks like Prism JS for the syntax highlighter does not support IE9:

https://github.com/PrismJS/prism/issues/987

There was a merge into Prism from the above thread, but when comparing the merge to the actual file in our node_modules folder for Prism JS, the code there doesn't match. Perhaps the npm version of prism is slightly out of date? I tried updating the npm version, but it looks like we're already using the latest.

The Prism Merge: https://github.com/PrismJS/prism/commit/c9bdcd3596380ecf572c9d2859e4191aec24463d

Our Version:

```JS
	if (document.addEventListener && !script.hasAttribute('data-manual')) {
		if(document.readyState !== "loading") {
			requestAnimationFrame(_.highlightAll, 0);
		}
		else {
			document.addEventListener('DOMContentLoaded', _.highlightAll);
		}
	}
```

Will hold on this for now so we can discuss options for approaching this. cc: @fuhton

**Comment by fuhton** [comment]
@RachelRVasquez Have we updated the version in `package.json`?

**Comment by RachelRVasquez** [comment]
@fuhton Yes I've tried to update it but we're already using the latest version. I meant that something with the npm version doesn't match what's in the Prism JS from their site, in spite of the npm version being up to date.

**Comment by fuhton** [comment]
Let's talk about this some more @RachelRVasquez I'm curious what your solution is.
